lsdcServer

```diff
@@ -1,3 +1,3 @@
 dzdo pkill -KILL -f daq_main2.py
-source /opt/conda_envs/lsdc-server-2022-1-latest/bin/activate
```

Comment:
The one in daq_mainAux is 2.2 and this one is 2.3. Should we go back to using "latest" to solve this and prevent code updates for conda env changes?

Reply:
let's just keep it as specific versions (2.2/2.3) for now. we may have a better deployment method for conda envs that creates the symlink automatically in the future, which would help us automate things more. I don't like having to make the symlink manually as it is right now